enhancements/sig-policy/28-selective-policy-enforcment/README.md

```diff
@@ -87,50 +87,208 @@ Out of scope for this feature
+remains in `inform` mode for clusters C and D.
 
+#### Example 2: No sub filtering
 ```
-metadata:
-  annotations:
-    versioned-policy-enable: [any|<generation>]
+kind: PlacementBinding
+placementRef:
+  apiGroup: apps.open-cluster-management.io
+  kind: PlacementRule
+  name: placementrule-extended
+remediationActionOverride:
+    remediationAction: enforce
+    subFilter: false
+subjects:
+- apiGroup: policy.open-cluster-management.io
+  kind: Policy
+  name: test-policy-1
 ```
 
-The values of the annotation govern how the controller will handle
-modifications to the parent policy specification while the child
-policy is in enforcing mode.
-
-The values of the annotation can take either of these forms:
-+ The value of `any` indicates that changes to the parent policy are
-immediately passed through to the enforcement of the child policy. As
-long as this annotation has the value of `any` remediation is enabled
-on the child policy and changes to the parent policy are immediately
-enforced to the cluster.
-+ An integer `generation` value indicates that the child policy will
-continue enforcing as long as the child policy `metadata/generation`
-value is equal to the specified value. In this way the user of this
-feature has control over whether changes to the parent policy happen
-immediately or require additional intervention by the user. At the
-expense of this additional complexity the behavior closes race
-conditions on a user/controller enabling/disabling enforcement and
-near-concurrent changes to the parent policy specification.
-
-The overall pattern follows a similar one in OLM operator updates
-where a manual installPlanApproval is explicitly enabled by the user
-when an update is desired.
-
-The implementation is relatively simple and defers the “selection”
-flexibility to the user/orchestrator/controller
+The `subFilter` field is false so all clusters in all placementrules
+which are bound to the policy are considered. In this case the bound
+PlacementRules are placementrule-initial and placementrule-extended
+(in this binding).
+
+Clusters A, B, C, D, E, and F are all set to enforce mode.
+
+#### Example 3: With sub filtering
+```
+kind: PlacementBinding
+placementRef:
+  apiGroup: apps.open-cluster-management.io
+  kind: PlacementRule
+  name: placementrule-extended
+remediationActionOverride:
+    remediationAction: enforce
+    subFilter: true
+subjects:
+- apiGroup: policy.open-cluster-management.io
+  kind: Policy
+  name: test-policy-1
+```
+
+The `subFilter` field is true in this example so only clusters in
+placementrules with `subFilter` that is false are considered. In this
+case that is only the placementrule-initial with clusters A, B, C, and
+D. The PlacementRule in this CR is `placementrule-extended` which
+includes A, B, E, and F.
+
+The set of clusters set to enforce mode is the intersection of these
+two sets. Clusters A and B are set to enforce mode.
```

Comment:
So E and F will still be bound without action override? Not sure in what use case this kind of mixed  binding behavior would be useful. For our main use case from TALM, it would make more sense to keep the subfilter at the binding level so that the new binding only does action override to already bound clusters and no other clusters get impacted.